src/Latte/Macros/BlockMacros.php

```diff
@@ -125,6 +129,13 @@ public function macroInclude(MacroNode $node, PhpWriter $writer)
 		} else {
 			$cmd = 'Latte\Macros\BlockMacrosRuntime::callBlock' . ($parent ? 'Parent' : '') . "(\$_b, $name, %node.array? + " . ($parent ? 'get_defined_vars' : '$template->getParameters') . '())';
 		}
+		$context = $this->getCompiler()->getContext() ? : array(NULL, NULL);
```

Comment:
I'm not sure about this. Is there any difference between `getContext() = NULL` and `getContext() = array(NULL, NULL)`?